plugin/gv.vim

```diff
@@ -211,11 +211,8 @@ function! s:setup(git_dir, git_origin)
   let b:git_dir = a:git_dir
 endfunction
 
-function! s:git_dir()
-  if empty(get(b:, 'git_dir', ''))
-    return fugitive#extract_git_dir(expand('%:p'))
-  endif
-  return b:git_dir
+function! s:git_dir() abort
+  return FugitiveGitDir()
```

Comment:
Thanks. We can just replace `s:git_dir()` calls with `FugitiveGitDir()` and remove the function.

Reply:
You mean like this ?
```
  let git_dir = FugitiveGitDir()
```
ok, sure, hang on.

Reply:
or do you mean remove it as an arg to s:setup() and everywhere else also ?